### ISSUE TITLE
Support for max_throughput to be lowered

### ISSUE THREAD

**Issue by markoctg** [hashicorp/terraform-provider-azurerm#16379]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

[Documentation](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cosmosdb_sql_database#max_throughput) states that the lowest possible value for CosmosDB max_throughput can be se to 4000 RU/s. Not sure why and who is imposing this lower limit, since on Azure portal I can manually set the value to as low as 1000 RU/s.

This should probably be resolved just by lowering or removing this limit.

### New or Affected Resource(s)/Data Source(s)

azurerm_cosmosdb_sql_database

### Potential Terraform Configuration

```hcl
resource "azurerm_cosmosdb_sql_database" "database" {
  name                = "SomeName"
  resource_group_name = "sample-cdb-rg"
  account_name        = "sample-cdb"
  autoscale_settings {
    max_throughput = 1000
  }
}
```

### References

No other references to this issue were found

**Comment by sinbai** [comment]
@markoctg thank you for posting this issue here.  I assume that it is a documentation issue as the issue has been fixed by PR [#15978](https://github.com/hashicorp/terraform-provider-azurerm/pull/15978). Also, I have submitted PR [#16387 ](https://github.com/hashicorp/terraform-provider-azurerm/pull/16387) to fix the documentation issue.

**Comment by mbfrahry** [comment]
Thanks @sinbai! @markoctg, the docs should reflect the correct value now. The old minimum value used to be 4000 but has since been lowered to 1000 in 3.x of the provider.
